src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2126,6 +2126,15 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                                     ->end()
                                     ->prototype('scalar')->end()
                                 ->end()
+                                ->arrayNode('recipients_allowed')
+                                    ->info('A list of regular expressions that allow recipients when "recipients" option is defined.')
+                                    ->performNoDeepMerging()
+                                    ->beforeNormalization()
+                                    ->ifArray()
+                                        ->then(fn ($v) => array_filter(array_values($v)))
```

Comment:
shouldn't this do `array_values` after filtering if you want to return a list ? Otherwise, I don't see the benefit.

Reply:
Actually, I copied what is done on `->arrayNode('recipients')`